packages/boba/contracts/contracts/LP/L1LiquidityPool.sol

```diff
@@ -209,6 +221,7 @@ contract L1LiquidityPool is CrossDomainEnabledFast, ReentrancyGuardUpgradeable,
         senderMessenger = _l1CrossDomainMessenger;
         relayerMessenger = _l1CrossDomainMessengerFast;
         L2LiquidityPoolAddress = _L2LiquidityPoolAddress;
+        L1StandardBridgeAddress = _L1StandardBridgeAddress;
```

Comment:
would it be better if you used the address manager?

Reply:
Actually, we can get all parameters from the address manager. However, it's dangerous to remove those variables and use the address manager to get all fixed addresses. To keep the consistence of the contract, I used the L1StandardBridgeAddress instead of the address manager. L1StandardBridgeAddress is fixed and we will never change it, so it's probably ok to set it as a constant variable.